packages/lib/server/service/replexica.ts

```diff
@@ -27,10 +28,36 @@ export class ReplexicaService {
 
       return result;
     } catch (error) {
+      logger.error(`ReplexicaService.localizeText() failed: ${error}`);
       return text;
     }
   }
 
+  /**
+   * Localize a text string to multiple target locales
+   * @param text The text to localize
+   * @param sourceLocale The source language locale
+   * @param targetLocales Array of the target language locales
+   * @returns The localized texts
+   */
+  static async batchLocalizeText(
+    text: string,
+    sourceLocale: string, // TODO: set type to LocaleCode once Replexica team exports it
+    targetLocales: string[] // TODO: set type to LocaleCode[] once Replexica team exports it
+  ): Promise<string[]> {
+    try {
+      const result = await this.engine.batchLocalizeText(text, {
+        sourceLocale: sourceLocale as any, // TODO: remove casting once TODO above is done
+        targetLocales: targetLocales as any, // TODO: remove casting once TODO above is done
```

Comment:
batchLocalizeText requires the type to be `LocaleCode` which isn't being exported by Replexica. Reported to Max.